docker/encfs/encfs.sh

```diff
@@ -22,11 +22,56 @@ fi
 
 echo CmdlineArgs = $CmdlineArgs
 
+cleanup_unmount() {
+  mount_array=$(process_encoded_string $EncfsSideCarArgs)
+  echo "Cleaning up by unmounting the following remote filesystem folder."
+  for mount_point in "${mount_array[@]}"; do
+    echo "attempting to unmount $mount_point"
+    output=$(umount $mount_point 2>&1)
+    exit_status=$?
+    echo "$output"
+    if [ $exit_status -eq 0 ]; then
+      echo "Unmounting was successful."
+    else
+      echo "Unmounting failed with exit status $exit_status."
```

Comment:
Should we not terminate here?

Reply:
So if unmounting the first mounted folder failed, we should continue to try to unmount other folders.